src/main/java/de/embl/cba/mobie/ui/viewer/SourcesPanel.java

```diff
@@ -120,8 +121,6 @@ private void addColorButton( JPanel panel, int[] buttonDimensions, SourceAndMeta
 
     private void setSourceColor( SourceAndMetadata< ? > sam, Color color, JPanel panel )
     {
-        sam.metadata().color = color.toString();
```

Comment:
Why don't you change this field anymore?

Reply:
I changed this because the output from color.toString() can't be read when this metadata is loaded again. Currently, this means that if you add an image (say one of the prospr genes), change the colour, remove it, and then re-add it again, it will appear white. This is why I added support for a "(r=255,g=255,b=255,a=255)" format in imagej-utils, and this is the version I save to the bookmarks. This does mean that colours can't persist though...

Reply:
I think you have to explain me in person... :-)